### ISSUE TITLE
New-Object error

### ISSUE THREAD

**Issue by shicarasmith** [brianbar-MSFT/Install-MMA#2]
I am getting an error with this script don't know if its still valid.

New-Object: Retrieving the COM class factory for component with CLSID {00000000-0000-0000-0000-000000000000} failed due to the following error: 80040154 Class not registered (Exception from HRESULT: 0x80040154 (REGDB_E_CLASSNOTREG)).        
InvalidOperation: You cannot call a method on a null-valued expression.
InvalidOperation: You cannot call a method on a null-valued expression.

**Comment by madshaun1984** [comment]
I'm also getting this, currently scratching my head as to why...

**Comment by shicarasmith** [comment]
> I'm also getting this, currently scratching my head as to why...

Yeah I spent hours last night trying to fix it.  All I can think of is this is happening at the very end.  I've tried a lot.

**Comment by madshaun1984** [comment]
In my case, I've solved it. It was failing to copy the MSI to the target server...

Copying the msi over manually then running the script again works. So the my issue is an account permission missing somewhere for the target server.
